From 280295377891f4cd6675938fa517589a411ba3ec Mon Sep 17 00:00:00 2001
From: Jajauma <jajauma@2ch.hk>
Date: Sat, 30 May 2015 20:14:49 +0300
Subject: [PATCH] Fix xdgZeroMemory fallback

---
 src/basedir.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/basedir.c b/src/basedir.c
index 68ab879..85eff27 100644
--- a/src/basedir.c
+++ b/src/basedir.c
@@ -61,7 +61,7 @@
 #else
 static void xdgZeroMemory(void* p, int n)
 {
-	while (n > 0) { ((char*)p)[n] = 0; ++n; }
+	while (n--) { ((char*)p)[n] = 0; }
 }
 #endif
 
-- 
1.8.3.1

